docs/project_defn.rst

```diff
@@ -287,7 +286,9 @@ using `GCP Batch <https://cloud.google.com/batch>`_ and `Cloud Run <https://clou
        may help. Default: 40 MiB
 
 *  ``region``: The GCP region in which the job will be run and the region of the Artifact Registry.
-*  ``batch_array_size``: Number of tasks to divide the simulations into. Max: 10000.
+   (e.g. ``us-central1``)
+*  ``batch_array_size``: Number of tasks to divide the simulations into. Tasks with fewer than 100
+   simulations each are recommended, especially when using spot instances. Max: 10,000.
```

Comment:
to avoid loosing too much of simulations or anything in respect to that when preemption happens? maybe adding something like this to justify the recommendation on < 100 sims per task?

Reply:
Added.